src/components/interfaces/HMI_API.xml

```diff
@@ -2267,6 +2270,25 @@
       </param>
   </struct>
 
+  <struct name="SpatialStruct">
```

Comment:
Alright, with the acceptance of https://github.com/smartdevicelink/sdl_evolution/pull/254, a number of changes were made to this struct.